community-website/src/community-project-boilerplate-docgen/layouts/common/header.pug

```diff
@@ -1 +1,7 @@
 div !{header}
+  .marketing-banner
+    .banner__description
+      a(href='https://www.algolia.com/doc/guides/building-search-ui/what-is-instantsearch/angular/')
+        p
+          strong Angular InstantSearch V3 beta has been documented on algolia.com/doc, check it out →
```

Comment:
FYI we don't mention the word "beta" for the Vue website: https://v2--vue-instantsearch.netlify.com/getting-started/getting-started.html.

Reply:
would you just keep "V3" ?

Reply:
We do mention beta: https://community.algolia.com/vue-instantsearch/getting-started/getting-started.html

Reply:
ok, so do you guys think we should or shouldn't keep the beta in the name? (I prefer when we clearly mention it's a beta, users know to expect breaking changes)

Reply:
Beta or not beta, the user expects breaking changes since it's a new major.

Either way is fine. Note that once we release the stable, we need to remove the word "beta". Since this move is to "deprecate" the old doc, I figured we won't update the community website very often from now on.

@Haroenv You're pointing to the migration guide, not to the new doc.

Reply:
> Beta or not beta, the user expects breaking changes since it's a new major.
I meant breaking changes from 3.0.0-beta.0 and 3.0.0-beta.1 for example.

Reply:
Right! Anyway, we can add the word "beta" for now, so long as we don't forget to remove it afterwards.